File: frontend/src/api/ledgerBook.ts
export type LedgerBook = "PERSONAL" | "HOUSEHOLD";

export const LEDGER_BOOKS: { id: LedgerBook; label: string }[] = [
  { id: "PERSONAL", label: "개인" },
  { id: "HOUSEHOLD", label: "가계" },
];

const STORAGE_KEY = "pininicong.ledgerBook";

export function loadLedgerBook(): LedgerBook {
  const raw = localStorage.getItem(STORAGE_KEY);
  if (raw === "PERSONAL" || raw === "HOUSEHOLD") return raw;
  return "PERSONAL";
}

export function saveLedgerBook(book: LedgerBook): void {
  localStorage.setItem(STORAGE_KEY, book);
}

export function ledgerBookLabel(book: LedgerBook): string {
  return LEDGER_BOOKS.find((b) => b.id === book)?.label ?? book;
}

File: backend/src/main/java/com/pininicong/cashbook/domain/LedgerBook.java
package com.pininicong.cashbook.domain;

public enum LedgerBook {
    PERSONAL("개인"),
    HOUSEHOLD("가계");

    private final String label;

    LedgerBook(String label) {
        this.label = label;
    }

    public String label() {
        return label;
    }
}
